chrome/content/zotero/xpcom/data/item.js

```diff
@@ -3613,13 +3613,19 @@ Zotero.defineProperty(Zotero.Item.prototype, 'attachmentDataURI', {
 			return '';
 		}
 		let buf = await OS.File.read(path, {});
-		let bytes = new Uint8Array(buf);
-		let binary = '';
-		let len = bytes.byteLength;
-		for (let i = 0; i < len; i++) {
-			binary += String.fromCharCode(bytes[i]);
-		}
-		return 'data:' + this.attachmentContentType + ';base64,' + btoa(binary);
+		buf = new Uint8Array(buf).buffer;
+		// It's 10 times faster to use FileReader than to use btoa()
```

Comment:
Can remove these comments — don't need comments referring to removed code

Reply:
OK removed now